docs-client/src/components/VariableList/index.tsx

```diff
@@ -83,168 +90,136 @@ export default function({
   );
 }
 
-interface State {
-  isEmpty: boolean;
-  isBeans: boolean[];
-  openBeans: boolean[];
-  colSpanLength: number;
-  childStates: { [key: number]: State };
-}
-
 interface OwnProps {
   indent: number;
-  hasBean: boolean;
-  state?: State;
-  setChildStateInParent?: (index: number, childState: State) => void;
-  childIndex: number;
+  hasLocation: boolean;
+  hidden?: boolean;
 }
 
 type FieldInfosProps = OwnProps & Props;
 
-class FieldInfos extends React.Component<FieldInfosProps, State> {
-  private static generateKey(pre: string): string {
-    return `${pre}_${new Date().getTime()}`;
-  }
+interface FieldInfoProps {
+  hasLocation: boolean;
+  hidden?: boolean;
+  indent: number;
+  variable: Variable;
+  specification: Specification;
+  title: string;
+}
 
-  private static indentString(indent: number, s: string): string {
-    return `${'\xa0'.repeat(indent)}${s}`;
-  }
+const indentString = (indent: number, s: string): string => {
+  return `${'\xa0'.repeat(indent)}${s}`;
+};
 
-  private static formatLocation(s: string): string {
-    return this.formatRequirement(s);
+const formatRequirement = (s: string): string => {
+  const lowerCase = s.toLowerCase();
+  if ('unspecified' === lowerCase) {
+    return '-';
   }
 
-  private static formatRequirement(s: string): string {
-    const lowerCase = s.toLowerCase();
-    if ('unspecified' === lowerCase) {
-      return '-';
-    }
+  return lowerCase;
+};
 
-    return lowerCase;
-  }
+const formatLocation = (s: string): string => {
```

Comment:
nit: After this function is migrated to lambda, I think this works for this (alias|wrap) case. 😉
``` ts
const formatLocation = formatRequirement
```